gui/pages/Content/Models/MarketModels.js

```diff
@@ -0,0 +1,54 @@
+import React, {useState, useEffect} from "react";
+import styles from "@/pages/Content/Marketplace/Market.module.css";
+import Image from "next/image";
+import {loadingTextEffect, returnToolkitIcon} from "@/utils/utils";
+import {EventBus} from "@/utils/eventBus";
+import {fetchMarketPlaceModel} from "@/pages/api/DashboardService";
+
+export default function MarketModels(){
+    const [showMarketplace, setShowMarketplace] = useState(false);
+    const [isLoading, setIsLoading] = useState(false)
+    const [loadingText, setLoadingText] = useState("Loading Models");
+    // const modelTemplates = [{'name':'model 1','description':'shifting timeline across multiple time strings. Regard shifting multiple time string is the agents to be.','provider':'Replicate'},
+    //     {'name':'model 1','description':'shifting timeline across multiple time strings. Regard shifting multiple time string is the agents to be.','provider':'Replicate'},
+    //     {'name':'model 1','description':'shifting timeline across multiple time strings. Regard shifting multiple time string is the agents to be.','provider':'Replicate'},{'name':'model 1','description':'shifting timeline across multiple time strings. Regard shifting multiple time string is the agents to be.','provider':'Replicate'},
+    //     {'name':'model 1','description':'shifting timeline across multiple time strings. Regard shifting multiple time string is the agents to be.','provider':'Replicate'},]
+    const [modelTemplates, setModelTemplates] = useState([]);
+
+    useEffect(() => {
+        loadingTextEffect('Loading Models', setLoadingText, 500);
+    },[]);
+
+    useEffect(() => {
+        fetchMarketPlaceModel().then((response) => {
+            console.log(response.data)
+            setModelTemplates(response.data)
+        })
+    },[])
+
+    function handleTemplateClick(item) {
+        const contentType = 'model_template';
```

Comment:
don't use temporary variables, write this directly in the EventBus

Reply:
Sir the value cant we used directly in the Eventbus. Either the structure should be key:value pair or the use of temp variables.